File: https-quera.org-problemset-51865/Solution.java
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        app();
    }


    public static void app() {
        Scanner scanner = new Scanner(System.in);

        int res = scanner.nextInt();

        int days = scanner.nextInt();

        if (days == 0) {
            res = 20;
        } else if (days != 7) {
            res -= days;
        }

        if (res < 0) {
            res = 0;
        }

        System.out.println(res);
    }

}

File: https-quera.org-problemset-51865/Solution.py
def app():
    res = int(input())
    days = int(input())

    if days == 0:
        res = 20

    elif days != 7:
        res -= days

    if res < 0:
        res = 0

    print(res)

if __name__ == "__main__":
    app()